docs/conf.py

```diff
@@ -60,11 +60,18 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-#
+
+
+def extract_version():
+    with open("../pyproject.toml") as f:
+        parsed_toml = toml.loads(f.read())
+    return parsed_toml["tool"]["poetry"]["version"]
+
+
 # The short X.Y version.
-version = __version__
+version = extract_version()
 # The full version, including alpha/beta/rc tags.
-release = __version__
+release = extract_version()
 
```

Comment:
Reading from disk and parsing toml file twice.
This `__version__ = pkg_resources.get_distribution("nornir").version` didn't work?